api/test/e2e/global_rule_test.go

```diff
@@ -234,7 +255,7 @@ func TestGlobalRule(t *testing.T) {
 			Object:        APISIXExpect(t),
 			Method:        http.MethodGet,
 			Path:          "/hello",
-			Query:         "name=;select%20from%20sys",
+			Query:         "file=root.exe",
```

Comment:
What's this? 😳

Reply:
it's query string for `uri-blocker` https://github.com/apache/apisix/blob/master/doc/plugins/uri-blocker.md